spice_ev/strategies/distributed.py

```diff
@@ -20,14 +19,67 @@ def __init__(self, components, start_time, **kwargs):
         strat_options_deps = dict(deepcopy(kwargs), **strat_options_deps)
         self.description = f"distributed {strat_opps}/{strat_deps}"
         self.strat_opps = strategy.class_from_str(strat_opps)(
-            components, start_time, **strat_options_opps)
+            comps, start_time, **strat_options_opps)
         self.strat_deps = strategy.class_from_str(strat_deps)(
-            components, start_time, **strat_options_deps)
+            comps, start_time, **strat_options_deps)
 
+        # adjust foresight for vehicle events (known one hour in advance)
+        self.A_HORIZON = datetime.timedelta(hours=1)
         # minimum charging time at depot; time to look into the future for prioritization
-        self.C_HORIZON = 3  # min
-        # dict that holds the current vehicles connected to a grid connector for each gc
-        self.v_connect = {gc_id: [] for gc_id in self.world_state.grid_connectors.keys()}
+        self.C_HORIZON = datetime.timedelta(minutes=3)
```

Comment:
Maybe use `ARRIVAL_HORIZON` and `CHARGING_HORIZON` / `PRIORIZATION_HORIZON` to make it clearer what this represents?